spring-boot-starter/starter/pom.xml

```diff
@@ -4,6 +4,7 @@
     <groupId>org.camunda.bpm.springboot.project</groupId>
     <artifactId>camunda-bpm-spring-boot-starter-root</artifactId>
     <version>7.16.0-SNAPSHOT</version>
+    <relativePath>../</relativePath>
```

Comment:
Did you receive a warning anywhere to add this? As far as I know, that is the default for Maven anyways.

Reply:
Good point. I wasn't aware of the default value. At one point I had the impression that my changes in the parent poms were not picked up, so I added this row and it resolved, but I think I misunderstood that.